src/spotlight.cc

```diff
@@ -87,15 +88,35 @@ namespace {
 Spotlight::Spotlight(QObject* parent)
   : QObject(parent)
   , m_activeTimer(new QTimer(this))
+  , m_clickTimer(new QTimer(this))
+  , m_virtualDevice(new VirtualDevice)
 {
   m_activeTimer->setSingleShot(true);
   m_activeTimer->setInterval(600);
+  m_clickTimer->setSingleShot(true);
+  m_clickTimer->setInterval(dblClickDuration);
+
+  if (!m_virtualDevice->isDeviceCreated()){
+      char msg[] = "A virtual device was not created. "\
+                   "Some features like changing pointer modes might not work.\n\n"\
+                   "Please check if uinput kernel module is loaded.";
```

Comment:
This won't be picked up by Qt's `lupdate` tool. 
Use `const auto msg = tr("...");` and then the `msg` variable directly for QMessagebox